checkstyle.xml

```diff
@@ -52,16 +52,6 @@ page at http://checkstyle.sourceforge.net/config.html -->
       <property name="severity" value="error"/>
     </module>
 
-    <module name="ImportOrder">
```

Comment:
I need to check whether the problem is this check or that files aren't following these rules.

Reply:
Rules are:

.3.3 Ordering and spacing

Imports are ordered as follows:

    All static imports in a single block.
    All non-static imports in a single block.

If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.

Within each block the imported names appear in ASCII sort order. (Note: this is not the same as the import statements being in ASCII sort order, since '.' sorts before ';'.)

Reply:
Done.